pkgs/by-name/te/textlint-rule-prh/test.md

```diff
@@ -0,0 +1 @@
+オープンソースソフトウェア
```

Comment:
no need to change, but you can inline this

Reply:
A separate file is also helpful for quick debugging.